lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -133,8 +134,14 @@ class EffectiveClangContext {
     return StringRef(Unresolved.Data, UnresolvedLength);
   }
 };
+
+#if LLVM_PTR_SIZE == 4
+static_assert(sizeof(EffectiveClangContext) <= 4 * sizeof(void *),
+              "should fit in four pointers");
+#else
 static_assert(sizeof(EffectiveClangContext) <= 2 * sizeof(void *),
               "should fit in a couple pointers");
+#endif
```

Comment:
I wonder if we can do something more generic considering that `LLVM_PTR_SIZE` is equal to `sizeof(void *)` and we know that `EffectiveClangContext` should be less than or equal to 16 bytes long. Otherwise, LGTM. Nice catch!